server/views/pages/licence/partials/induction.njk

```diff
@@ -0,0 +1,17 @@
+{% macro induction(person, address, timeOfMeeting, contactNumber) %}
+    <div class="section" id="induction">
+        On release from prison (including, if applicable, any release from from detention under the Immigration Act 1971
+        during the currency of your licence, whether or not leave has been granted for you to remain in the United Kingdom),
+        unless otherwise directed by your supervising officer, you must report without delay to:
+        <div id="meeting-details">
+            <p>
+                <span class="bold">{{ person or "Duty officer" }}</span><br/>
+                {% for a in (address | formatAddressAsList) %}
+                    <span class="bold">{{ a }}</span><br>
```

Comment:
`<br>` should not be used. Give your span this css instead

```
{
    display: block;
    margin-bottom: 10px;
}
```

Reply:
Or use a `<p>` tag 😄

Reply:
Will address

Reply:
Done this now - coming on next PR.